ruleset.xml

```diff
@@ -1,23 +1,5 @@
 <?xml version="1.0"?>
 <ruleset name="Zend Framework Coding Standard">
     <description>Zend Framework Coding Standard</description>
-
-    <!-- display progress -->
-    <arg value="p"/>
-    <arg name="colors"/>
-
-    <!-- inherit rules from: -->
-    <rule ref="PSR2"/>
-    <rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
-    <rule ref="Generic.Formatting.SpaceAfterNot"/>
-    <rule ref="Squiz.WhiteSpace.OperatorSpacing">
-        <properties>
-            <property name="ignoreNewlines" value="true"/>
-        </properties>
-    </rule>
-    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
-        <properties>
-            <property name="ignoreBlankLines" value="false"/>
-        </properties>
-    </rule>
+    <rule ref="vendor/zendframework/zend-coding-standard/ZendCodingStandard/ruleset.xml"/>
```

Comment:
A little confused by the changes here.

- In components that already use this package, this particular file is the one referenced. As an example, from zendframework/zend-expressive:

  ```xml
  <rule ref="./vendor/zendframework/zend-coding-standard/ruleset.xml"/>
  ```

  Shouldn't this then be the full ruleset, and not reference another ruleset?

- The referenced path would be potentially valid for components referencing the ruleset installed via Composer, _assuming that phpcs follows the reference_, but is malformed; `ZendCodingStandard` is a subdirectory of `src/`~

- The referenced path is invalid for _this_ repository (should be `src/ZendCodingStandard/ruleset.xml`), though I'm not sure if that's a problem or not.

I suspect the testing against other components will tell us for sure if this works as expected, but thought I'd highlight the potential issues first.